subprojects/android-test/android-log/src/main/kotlin/com/avito/android/log/sentry/SentryDestination.kt

```diff
@@ -0,0 +1,40 @@
+package com.avito.android.log.sentry
+
+import com.avito.logger.LogLevel
+import com.avito.logger.LoggingDestination
+import io.sentry.SentryClient
+import io.sentry.SentryClientFactory
+
+class SentryDestination(private val dsn: String) : LoggingDestination {
+
+    @Transient
+    private lateinit var _sentry: SentryClient
+
+    private fun createSentry(): SentryClient {
```

Comment:
```suggestion
    private fun sentry(): SentryClient {
```
Otherwise, it looks like the method creates an instance on each invocation.